src/NorthstarClient.php

```diff
@@ -54,7 +39,7 @@ public function verify($credentials)
      */
     public function getAllUsers($inputs = [])
     {
-        $response = $this->get('users', $inputs);
+        $response = $this->get('v1/users', $inputs);
```

Comment:
Not sure, but would it make sense at all to store the version number as a property or config setting somewhere?